src/components/hue/hue.component.tsx

```diff
@@ -9,9 +9,10 @@ import { Interactive } from "../interactive";
 interface IHueProps {
   readonly color: IColor;
   readonly onChange: (color: IColor) => void;
+  readonly disabled: boolean;
```

Comment:
```suggestion
  readonly disabled?: boolean;
```

`disabled` props should optional for all components as they might be used detached from `ColorPicker`.